lib/onc_certification_g10_test_kit/onc_program_procedure.yml

```diff
@@ -259,7 +259,7 @@ procedure:
             not include support for “plain”
 
           Additionally, the following “capabilities” must be supported if using
-          US Core 5.0.1 or 6.1.0:
```

Comment:
Has this been validated against the test procedure (wording needs to be identical)